tools/packaging/kata-deploy/helm-chart/kata-deploy/values.yaml

```diff
@@ -1,48 +1,270 @@
 imagePullPolicy: Always
+
 imagePullSecrets: []
+
 image:
   reference: quay.io/kata-containers/kata-deploy
   tag: ""
-# k8s-dist can be k8s, k3s, rke2, k0s, microk8s
-k8sDistribution: "k8s"
+
+k8sDistribution: "k8s"  # k8s, k3s, rke2, k0s, microk8s
+
 # Node selector to control which nodes the kata-deploy daemonset runs on
 # Example:
 # nodeSelector:
 #   kata-containers: "enabled"
 #   node-type: "worker"
 nodeSelector: {}
 
-# RuntimeClass configuration
-# When enabled, RuntimeClasses will be created by the Helm chart instead of by the kata-deploy script
+debug: false
+
+snapshotter:
+  setup: []  # ["nydus", "erofs"] or []
+
+# See MAINTENANCE.md for field descriptions and maintenance guide
+# NOTE: All shims are disabled by default. Enable the ones you need explicitly.
+shims:
+  clh: # cloud-hypervisor, golang runtime
+    enabled: false
+    supportedArches:
+      - amd64
+      - arm64
+    allowedHypervisorAnnotations: []
+    containerd:
+      snapshotter: ""
+  cloud-hypervisor: # rust runtime
+    enabled: false
+    supportedArches:
+      - amd64
+      - arm64
+    allowedHypervisorAnnotations: []
+    containerd:
+      snapshotter: ""
+  dragonball: # rust runtime
+    enabled: false
+    supportedArches:
+      - amd64
+      - arm64
+    allowedHypervisorAnnotations: []
+    containerd:
+      snapshotter: ""
+  
+  fc: # firecracker, golang runtime
+    enabled: false
+    supportedArches:
+      - amd64
+      - arm64
+    allowedHypervisorAnnotations: []
+    containerd:
+      snapshotter: "devmapper" # requires pre-configuration on the user side
+  firecracker: # firecracker, golang runtime
```

Comment:
Is firecracker the runtime-rs version? I can't remember seeing that runtimeclass around though?

Reply:
I removed it, as we indeed don't have any runtimeclass.